tests/vmipreset_test.go

```diff
@@ -382,6 +440,6 @@ func waitForPreset(virtClient kubecli.KubevirtClient, prefix string) v1.VirtualM
 			}
 		}
 		return false
-	}, time.Duration(60)*time.Second).Should(Equal(true), "Timed out waiting for preset to appear")
+	}, time.Duration(60)*time.Second).Should(Equal(false), "timed out waiting for preset to be deleted")
```

Comment:
we don't have to perform a list on all the VMPresets to see that the preset no longer exists. 

```
_, err = virtClient.VirtualMachineInstancePreset(namespace).Get(presetName, &k8sv1.GetOptions{})
if errors.IsNotFound(err) {
    return false
}
return true
```

Reply:
This is true. you know this was copy/pasted from the original... ;) I'll fix it.

Reply:
fixed. thanks.

Reply:
yup, makes sense.